package-lock.json

```diff
@@ -419,6 +419,7 @@
             "has": "^1.0.3",
```

Comment:
Please revert these changes. (`git checkout main package-lock.json`)

Reply:
I'll revert this